playbooks/build-oc-images.yml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
I believe this playbook doesn't fit to provision/install workflows and should be used only for building images, not actual deployments - therefore it shouldn't be included in CLI utils.